src/main/java/org/akaza/openclinica/controller/ScheduledExtractController.java

```diff
@@ -0,0 +1,124 @@
+package org.akaza.openclinica.controller;
+
+import core.org.akaza.openclinica.bean.extract.ArchivedDatasetFileBean;
+import core.org.akaza.openclinica.bean.login.UserAccountBean;
+import core.org.akaza.openclinica.dao.extract.ArchivedDatasetFileDAO;
+import core.org.akaza.openclinica.dao.hibernate.StudyDao;
+import core.org.akaza.openclinica.domain.datamap.Study;
+import core.org.akaza.openclinica.service.UtilService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.apache.http.entity.ContentType;
+import org.quartz.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.HttpHeaders;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+
+
+@Controller
+@RequestMapping(value = "/auth/api")
+@Api(value = "ScheduledExtractJob", tags = {"Scheduled Extract Job"}, description = "REST API for Scheduled Extract Job")
+public class ScheduledExtractController {
+
+    private final static Logger logger = LoggerFactory.getLogger(ScheduledJobController.class);
+
+    @Autowired
+    private UtilService utilService;
+
+    @Autowired
+    private Scheduler scheduler;
+
+    @Autowired
+    private StudyDao studyDao;
+
+    @Autowired
+    private ArchivedDatasetFileDAO archivedDatasetFileDAO;
+
+
+    @ApiOperation(value = "To get latest scheduled extract dataset ids and creation time for the job name at study level", notes = "only work for authorized users with the right access permission")
+    @RequestMapping(value = "/studies/{studyOID}/extractJobs/{jobName}/jobExecutions", method = RequestMethod.GET)
```

Comment:
if we are going to update all jobName references to jobUuid, maybe we should change this url to also use jobUuid?

Reply:
I think its better the change it when the references to it gets changed to job uuid too in OC-12828. I began to work on it and realized that there's a lot of references to be changed as well in all the job servlets.